### PR TITLE
Rad storms now cause maintenance doors to be all-access for a short time.

### PR DESCRIPTION
**Pull request by Kokojos**
Does what it says on the tin. ^

### CODE REVIEW

**Comment by Carbonhell** [comment]
literal cure for cancer,i love u

**Comment by Nebyab** [comment]
k

**Comment by calibraptor** [comment]
Yeah sure holy shit

**Comment by ghost** [comment]
woah yeah dude

**Comment by Badmex** [comment]
Uh yeah +1/3

File: html/changelogs/Kokojo - radicalstorms.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Kokojo
+
+delete-after: True
+
+changes:
+  - rscdel: "Maintenance doors now become all-access when radiation storm hits."
```
Comment (Carbonhell) [inline html/changelogs/Kokojo - radicalstorms.yml]: isn't it a rscadd you FUCK

**Comment by Carbonhell** [comment]
GOTTA MERGE FAST

**Pull request merged**